src/common/expression/ParameterExpression.h

```diff
@@ -0,0 +1,47 @@
+/* Copyright (c) 2021 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#pragma once
+
+#include "common/expression/Expression.h"
+
+// The ParameterExpression use for parameterized statement
+namespace nebula {
+
+class ParameterExpression : public Expression {
```

Comment:
I doubt if we need this one.

Reply:
Is need. 
Such as lookupValidators or some visitor or scenes that require expression rewriting.

Some Example :
  - rewrite ParameterExpression to ConstantExpression
  - EvaluableExprVisitor determines whether expressions are evaluable by their specific types. Code likes that:
  ```
void visit(VariableExpression *) override { isEvaluable_ = false; }
```
  - If `ParameterExpression` and `VariableExpression` are unified, then we must use if-else whenever we need to distinguish them, I don't think this is a good code design pattern.